node/core/dispute-coordinator/src/lib.rs

```diff
@@ -671,6 +770,9 @@ async fn handle_import_statements(
 
 	overlay_db.write_candidate_votes(session, candidate_hash, votes.into());
 
+	// TODO(ladi): should we respond with ImportStatementsResult::ValidImport?
+	//pending_confirmation.send(ImportStatementsResult::ValidImport) .map_err(|_| Error::OneshotSend)?;
```

Comment:
Question: We do not reply to the pending confirmation? Is this desired behavior?

In it's current form, handle_new_activations he caller will only ever receive a reply on an invalid import (see above). Should we not also reply with `ValidImport` upon success?

Reply:
Yeah, nice catch. Should be covered in tests as well.